src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -76,6 +77,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Update database tables related to mentor approval.
     addApproval(mentorId, approverId);
     addEvidence(isApproved, mentorId);
+
+    // If mentor review is complete, send them a notification.
+    if (Utility.isReviewed(true, mentorId) == Utility.MENTOR_APPROVED) {
```

Comment:
i know i've forgotten to do this, but adding a comment for the boolean parameter is always good to add readability, something like ```/*param=*/true```

Reply:
Done!